Controller/Admin/Transaction/TransactionTabPostProcessing.php

```diff
@@ -482,8 +482,14 @@ private function _getPaymentMethodConfig()
 
         if (!is_null($this->_oTransaction)) {
             $sPaymentId = $this->_oTransaction->getPaymentType();
-            $oPaymentMethod = PaymentMethodFactory::create($sPaymentId);
-            $oConfig = $oPaymentMethod->getConfig();
+
+            if ($sPaymentId) {
+                $oPaymentMethod = PaymentMethodFactory::create($sPaymentId);
+                $oConfig = $oPaymentMethod->getConfig();
+            } else { // "wiretransfer"
+                $oPaymentMethod = PaymentMethodFactory::create("wdpaymentoninvoice");
```

Comment:
Let's please add a **comment** explaining why it is required to create a POI payment method (not PIA or something else).

Reply:
👍